tails_files/install.sh

```diff
@@ -68,23 +87,360 @@ if [ -f $XSESSION_RC ]; then
+  cp -f "${securedrop_dotfiles}/document.desktop" "${tails_live_dotfiles}/Desktop"
+  cp -f "${securedrop_dotfiles}/source.desktop" "${tails_live_dotfiles}/Desktop"
+  cp -f "${securedrop_dotfiles}/document.desktop" "${tails_live_dotfiles}/.local/share/applications"
+  cp -f "${securedrop_dotfiles}/source.desktop" "${tails_live_dotfiles}/.local/share/applications"
+}
+
+function set_permissions_on_desktop_shortcuts()
+{
+  # Follow up function to enforce correct permissions and ownership
+  # on desktop shortcuts for Document and Source Interfaces. Separated
+  # into function to aid readability.
+  chown amnesia:amnesia \
+    "${amnesia_desktop}/document.desktop" \
+    "${amnesia_desktop}/source.desktop" \
+    "${amnesia_home}/.local/share/applications/document.desktop" \
+    "${amnesia_home}/.local/share/applications/source.desktop" \
+    "${tails_live_dotfiles}/.local/share/applications/document.desktop" \
+    "${tails_live_dotfiles}/.local/share/applications/source.desktop" \
+    "${tails_live_dotfiles}/Desktop/document.desktop" \
+    "${tails_live_dotfiles}/Desktop/source.desktop"
+
+  chmod 700 \
+    "${amnesia_desktop}/document.desktop" \
+    "${amnesia_desktop}/source.desktop" \
+    "${amnesia_home}/.local/share/applications/document.desktop" \
+    "${amnesia_home}/.local/share/applications/source.desktop" \
+    "${tails_live_dotfiles}/.local/share/applications/document.desktop" \
+    "${tails_live_dotfiles}/.local/share/applications/source.desktop" \
+    "${tails_live_dotfiles}/Desktop/document.desktop" \
+    "${tails_live_dotfiles}/Desktop/source.desktop"
+}
+
+function configure_network_manager_hook()
+{
+  # Install a persistent NetworkManager hook to write ATHS values to the
+  # system /etc/tor/torrc, so the Document Interface can be accessed via
+  # Tor Browser. On the Admin Workstation, the SSH ATHS values will also
+  # be added. The hook will run on "up" for any interface other than loopback,
+  # and is carefully ordered to run AFTER the Tails Tor hook and BEFORE the
+  # Tails Apt update hook.
+  if ! grep -q 'custom-nm-hooks' "${tails_live_persistence}/persistence.conf"; then
+    echo "/etc/NetworkManager/dispatcher.d	source=custom-nm-hooks,link" >> "${tails_live_persistence}/persistence.conf"
+  fi
+  mkdir -p "${tails_live_persistence}/custom-nm-hooks"
+
+  cp -f 65-configure-tor-for-securedrop.sh "${tails_live_persistence}/custom-nm-hooks"
+  cp -f 65-configure-tor-for-securedrop.sh "$network_manager_dispatcher"
+  chown root:root \
+    "${tails_live_persistence}/custom-nm-hooks/65-configure-tor-for-securedrop.sh" \
+    "${network_manager_dispatcher}/65-configure-tor-for-securedrop.sh"
+  chmod 755 \
+    "${tails_live_persistence}/custom-nm-hooks/65-configure-tor-for-securedrop.sh" \
+    "${network_manager_dispatcher}/65-configure-tor-for-securedrop.sh"
+
+  # Run the SecureDrop init script. Among other things, implements the torrc
```

Comment:
> Among other things

Actually, the _only_ thing the init script does is implement the torrc additions.